com.github.mightycreak.Diffuse.yml

```diff
@@ -0,0 +1,19 @@
+app-id: com.github.mightycreak.Diffuse
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: diffuse
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=home
+modules:
+  - name: diffuse
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/MightyCreak/diffuse
+        branch: v0.6.0
+rename-desktop-file: diffuse.desktop
+rename-icon: diffuse
```

Comment:
You probably want to rename those upstream

Reply:
I'm sorry, I'm new to flatpak packaging, I created the manifest by looking at manifest on other apps. How should I rename these?

Reply:
What he's saying is that you should change the names of those files in the application's source directly as opposed to making the Flatpak builder do it, since it appears to be your code (or at least your repository).

Reply:
Thanks @x-m7

Reply:
Ok, so I've looked a bit more at what all that is doing. The thing is, while it is my repository, it is a fork of a no longer maintained repo (on sourceforge).

I'm still learning a lot about quite all the concepts (Python, GTK, Meson, Flatpak, Flathub) on top of the application code and structure themselves, so I'd rather keep these "rename" properties for now as I don't have to do much changes in the existing repo files. (What I'm afraid is if my application will still work when installed without Flatpak if I rename the files upstream).

I'd like to be a bit more comfortable with all that before renaming the files, if it's ok with you?